src/digest/base.cr

```diff
@@ -0,0 +1,83 @@
+require "value_ref"
+require "base64"
+
+module Digest::Base(C)
```

Comment:
Why? Naming standards we all agree on (i.e. dont use 1 letter variable names) seem to get routinely ignored when using generics. Wouldn't `ContextType` be a better name? `T`  is a good generic name for generic containers, this is not a generic container.